src/controllers/collection.controller.ts

```diff
@@ -247,6 +247,7 @@
     cloudinary.upload(req.file, "collections", data.id);
   }
 
+  redis.del("collections", { exclude: "collections?id=" });
```

Comment:
Hmmm, can i do like instead of manually deleting cache, for all my get requests, the frontend can add header Cache-Control with no-cache => i will fetch data from db, then save it to cache, next requests will be cached, if user add, update or delete, frontend will jusst have to add header and i have to fix less code